packages/lesswrong/lib/collections/posts/fragments.ts

```diff
@@ -213,6 +213,26 @@ registerFragment(`
   }
 `)
 
+registerFragment(`
+  fragment PostsModerationGuidelines on Post {
```

Comment:
I made a new fragment for this as moderation guidelines are only actually used in one place on the frontend, and it already has its own `useMulti`, so we can save some bandwidth here.